### PR TITLE
Create Tafsir.net.xml (1 Merge = 2 PRs closed)

### PR DESCRIPTION
**Pull request by I-have-50-open-PRs-to-HTTPS-EVERYWHERE**
Ameliorating https://github.com/EFForg/https-everywhere/pull/11102

### CODE REVIEW

File: src/chrome/content/rules/Tafsir.net.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<ruleset name="Tafsir.net">
+	<target host="tafsir.net"/>
+	<target host="www.tafsir.net"/>
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Tafsir.net.xml]: api.tafsir.net

File: src/chrome/content/rules/Tafsir.net.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	<target host="cpanel.tafsir.net"/>
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Tafsir.net.xml]: library.tafsir.net
www.library.tafsir.net

File: src/chrome/content/rules/Tafsir.net.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	<target host="mlffat.tafsir.net"/>
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Tafsir.net.xml]: www.mlffat.tafsir.net

File: src/chrome/content/rules/Tafsir.net.xml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+	<target host="media.tafsir.net"/>
+	<target host="vb.tafsir.net"/>
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Tafsir.net.xml]: www.vb.tafsir.net

File: src/chrome/content/rules/Tafsir.net.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<ruleset name="Tafsir.net">
+	<target host="tafsir.net"/>
+	<target host="www.tafsir.net"/>
+	<target host="cpanel.tafsir.net"/>
+	<target host="mlffat.tafsir.net"/>
+	<target host="media.tafsir.net"/>
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/Tafsir.net.xml]: services.tafsir.net
static.tafsir.net
www.static.tafsir.net
up.tafsir.net
www.up.tafsir.net

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@gloomy-ghost Added.

**Comment by gloomy-ghost** [comment]
Thanks!

**Pull request merged**